packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -82,7 +82,7 @@ class InspectorPreferencesController extends DisposableController
       _customPubRootDirectories;
   ValueListenable<bool> get isRefreshingCustomPubRootDirectories =>
       _customPubRootDirectoriesAreBusy;
-  InspectorService get inspectorService =>
+  InspectorService? get inspectorService =>
```

Comment:
This won't change anything as the `as` will still throw if it is not null.
`InspectorService? get inspectorService => serviceManager.inspectorService;`
is probably what you want.